schemas/interface.ts

```diff
@@ -249,11 +255,11 @@ declare namespace Quizoot {
         /**
          * The quiz author's first name.
          */
-        firstName: string;
+        name: string;
         /**
          * The quiz author's last name. Is optional.
          */
-        lastName?: string;
+        surname?: string;
```

Comment:
why 😂

Reply:
- First it's not respecting the case style: `lastName` is camelCase while the others fields in the interface are snake_case.
- Second, `name` is shorter and prettier than `firstName`
- Finally, if you really prefer using last and first, then we should write instead `lastname` and `firstname`